CONTRIBUTING.md

```diff
@@ -103,15 +106,16 @@ conda install -c conda-forge black
 ```sh
 # Inspect a file
 black FILE.py
+
 # Inspect files in the current folder
 black .
 ```
 
 #### Flake8 linter
 
-Flake8 is a famous Python linting package that might be of help within your development environment: <https://pypi.org/project/flake8/>. It will help ensure the correct format of your code.
+[Flake8](https://pypi.org/project/flake8/) is a famous Python linting package that might be of help within your development environment. It will help ensure the correct format of your code.
```

Comment:
We can maybe add that without a correct format the tests will failed

Reply:
OK